test/cmd/images.sh

```diff
@@ -57,6 +62,13 @@ os::cmd::try_until_success 'oc get imagestreamtags wildfly:latest'
 os::cmd::expect_success_and_text "oc get imageStreams wildfly --template='{{ index .metadata.annotations \"openshift.io/image.dockerRepositoryCheck\"}}'" '[0-9]{4}\-[0-9]{2}\-[0-9]{2}' # expect a date like YYYY-MM-DD
 os::cmd::expect_success_and_text 'oc get istag' 'wildfly'
 
+# create an image stream and post a mapping to it
+os::cmd::expect_success 'oc create imagestream test'
+os::cmd::expect_success 'oc create -f test/fixtures/mysql-image-stream-mapping.yaml'
+os::cmd::expect_success_and_text 'oc get istag/test:new --template="{{ index .image.dockerImageMetadata.Config.Entrypoint 0 }}"' "docker-entrypoint.sh"
+os::cmd::expect_success_and_text 'oc get istag/test:new -o jsonpath={.image.dockerImageReference}' "/$project/test@sha256:b2f400f4a5e003b0543decf61a0a010939f3fba07bafa226f11ed7b5f1e81237"
```

Comment:
It requires a bit more bash magic - but yes.

On Thu, Jun 9, 2016 at 5:14 AM, Michal Minar <notifications@github.com>
wrote:

> In test/cmd/images.sh
> <https://github.com/openshift/origin/pull/9238#discussion_r66406949>:
>
> > @@ -57,6 +62,13 @@ os::cmd::try_until_success 'oc get imagestreamtags wildfly:latest'
> >  os::cmd::expect_success_and_text "oc get imageStreams wildfly --template='{{ index .metadata.annotations \"openshift.io/image.dockerRepositoryCheck\ <http://openshift.io/image.dockerRepositoryCheck%5C>"}}'" '[0-9]{4}\-[0-9]{2}\-[0-9]{2}' # expect a date like YYYY-MM-DD
> >  os::cmd::expect_success_and_text 'oc get istag' 'wildfly'
> >
> > +# create an image stream and post a mapping to it
> > +os::cmd::expect_success 'oc create imagestream test'
> > +os::cmd::expect_success 'oc create -f test/fixtures/mysql-image-stream-mapping.yaml'
> > +os::cmd::expect_success_and_text 'oc get istag/test:new --template="{{ index .image.dockerImageMetadata.Config.Entrypoint 0 }}"' "docker-entrypoint.sh"
> > +os::cmd::expect_success_and_text 'oc get istag/test:new -o jsonpath={.image.dockerImageReference}' "/$project/test@sha256:b2f400f4a5e003b0543decf61a0a010939f3fba07bafa226f11ed7b5f1e81237"
>
> Can we match docker registry url as well?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/9238/files/b7b4a40b35a57e46528ad9efb1f26e309b8bc8fd..7d726a83f37add9d855d96696a30d8e2d296a314#r66406949>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABG_p113JtPpFfiEI1oSGc_zvZJeYO7_ks5qJ9lYgaJpZM4IxjR3>
> .
>